ansible/roles/software/ckan/catalog/ckan-app/molecule/catalog-next/tests/test_default.py

```diff
@@ -54,7 +54,8 @@ def test_production_ini(host):
     assert production_ini.mode == 0o640
 
     assert production_ini.contains('ckan.plugins =.*datajson_harvest')
-    assert production_ini.contains('ckan.plugins =.*saml2')
+    # TODO just for sandbox-next
+    # assert production_ini.contains('ckan.plugins =.*saml2auth')
```

Comment:
??? We should have saml enabled in sandbox.

Reply:
Yes @adborden. But is now called _saml2auth_. The name for old version is _saml2_. We are not installing the same in all catalog-next environments. 
That's why I comment this out. I don't know how to test different plugin names for each environment.

Reply:
This is what I wanted to discuss with @avdata99. Testing for an extension being in the configuration for something that is currently configured (this PR implements it only for sandbox) doesn't make sense. Once this is tested in sandbox, a new PR should be made that sets saml2auth as the default for all environments and changes how we deploy/use it as the default, instead of dependent upon the environment.

Reply:
Yes @jbrown-xentity and a reminder that molecule is not testing **any** environment. The tests are testing the role, independent of configuration. If you want the assertion here to be testing saml2auth, change it in the test setup https://github.com/GSA/datagov-deploy/blob/develop/ansible/roles/software/ckan/catalog/ckan-app/molecule/catalog-next/molecule.yml#L114